src/pages/sicp/Sicp.tsx

```diff
@@ -40,7 +44,8 @@ const Sicp: React.FC = () => {
   const [data, setData] = useState(<></>);
   const [loading, setLoading] = useState(false);
   const [active, setActive] = useState('0');
-  const { section } = useParams<{ section: string }>();
+  const { param_lang, section } = useParams<{ param_lang: string; section: string }>();
```

Comment:
Use camelCase, not snake_case for variables


```suggestion
  const { param_lang: langParam, section } = useParams<{ param_lang: string; section: string }>();
```